packages/shell/src/lib/app/commands.ts

```diff
@@ -28,6 +29,9 @@ export function isCommand(value: unknown): value is Command {
     case "clear-authentication": {
       return true;
     }
+    case "toggle-charm-list": {
```

Comment:
nit: Maybe instead of toggle, this sets the value (and the individual views are responsible for setting the inverted state), keeping the commands stateless